packages/shared/components/organisms/ReceiveAddressSection.svelte

```diff
@@ -0,0 +1,83 @@
+<script lang="typescript">
+    import { appSettings } from '@core/app'
+    import { QR, Text } from 'shared/components'
+    import { AddressBox, OnboardingButton } from 'shared/components/atoms'
+    import { localize } from '@core/i18n'
+    import { selectedAccount } from '@core/account'
+    import { FontWeightText } from 'shared/components/Text.svelte'
+    import { openPopup } from '@lib/popup'
+
+    export let addressFontSize = 'base'
+
+    let addressBoxElement: AddressBox
+
+    $: recieveAddress = $selectedAccount.meta.publicAddresses[0].address
+    $: darkModeEnabled = $appSettings.darkMode
+
+    function handleSendClick() {
+        openPopup({
+            type: 'sendForm',
+            overflow: true,
+        })
+    }
+
+    function handleRecieveClick() {
+        addressBoxElement.copyAddress()
+    }
+</script>
+
+<receive-details class="flex flex-col space-y-6 justify-between cursor-pointer">
+    <OnboardingButton primaryText={'Send funds'} secondaryText="Send tokens to an address" onClick={handleSendClick} />
+    <div
+        class="flex flex-col px-4 py-4 space-y-6 rounded-xl"
+        class:darkmode={darkModeEnabled}
+        on:click={handleRecieveClick}
+    >
+        <Text type="h5" fontWeight={FontWeightText.semibold} classes="text-left"
+            >{localize('general.receiveFunds')}</Text
+        >
+        <QR data={recieveAddress} classes="w-3/5 h-auto" />
```

Comment:
My thoughts on the QR code:

The QR code in the Figma file is 128x128 but it is 171x171 in this PR. The 128x128 looks a little small without the copy to clipboard text but the 171x171 still looks too large. I added a scaleRatio prop of 0.8 to 0.9 locally since the `size` prop wasn't working and it looks much better.

Original => 171x171:
![image](https://user-images.githubusercontent.com/72045705/177661990-b867264a-bb53-4380-93f1-d2401a31ebea.png)

scale(0.85) => 145x145:
![image](https://user-images.githubusercontent.com/72045705/177662209-b22829c9-2e5f-42a9-83bf-8240671918ff.png)

`<QR data={recieveAddress} scaleRatio="0.85" classes="w-3/5 h-auto" />`
```
export let scaleRatio = 1
<svg 
        style="transform: {`scale(${scaleRatio})`};" 
        width={size} 
        height={size} 
        viewBox={`0 0 ${cells.length} ${cells.length}`} 
        class={classes}
>
```

Reply:
Not to sure, we have had complaints in the past that the qr code is too small to scan. I thought we would want to utilise the space more..

Reply:
It scans with my iPhone camera app at 0.85 size, but I can't check on android